app/src/main/java/com/bogdan/codeforceswatcher/fragment/UsersFragment.kt

```diff
@@ -52,12 +52,12 @@ class UsersFragment : android.support.v4.app.Fragment(), SwipeRefreshLayout.OnRe
 
     private fun initViews() {
         fab.setOnClickListener { startActivity(Intent(requireContext(), AddUserActivity::class.java)) }
-        swiperefresh.setOnRefreshListener(this)
+        swiperefreshUsers.setOnRefreshListener(this)
 
         userAdapter = UserAdapter(listOf(), requireContext())
-        rvMain.adapter = userAdapter
-        rvMain.layoutManager = LinearLayoutManager(requireContext())
-        rvMain.addOnScrollListener(object : RecyclerView.OnScrollListener() {
+        rvUsers.adapter = userAdapter
```

Comment:
If you have only one `RecyclerView` on the screen it's easier (and also used much more often) to call it just `recyclerView`.